Cognitive reflection test (CRT)

amirrr opened the pull request:
Added CRT-6 test

markwhiting commented:
I think this can go ahead and merge, unless I missing something @JamesPHoughton?

JamesPHoughton commented:
Looks good to me - I haven't used images in these surveys before, so once we have it merged (which should trigger a push to npmjs) we should check that the images are packaged up properly and distributed with the package. Do you have a project you're ready to import it into to test the full dev chain?

markwhiting commented:
Thanks James! Yes, we do. @amirrr want to go-ahead with that now?

amirrr commented:
Yes. I will add it.

JamesPHoughton commented:
Cool. Does this PR also close https://github.com/Watts-Lab/surveys/pull/161, and address issues reviewed there?

amirrr commented:
It does, but that was my mistake. I updated the wrong branch with refactors. I also just fixed the scoring function on the RMET branch to be the same as here.
